src/components/site/SubpageTabs.md

```diff
@@ -17,7 +17,23 @@ import Section from "../basic/containers/Section";
 Html Snippet:
 
 ```html
-<div class="dark">
-  <ul></ul>
+<div class="dg_subpage-tabs-container">
+  <div class="container">
+    <div class="dg_subpage-tabs" id="dg_tabs">
+      <button
+        class="dg_subpage-tabs-btn"
+        type="button"
+        id="dg_menu-trigger-btn"
+        aria-expanded="true"
+      >
+        <h2 class="dg_accordion_buttontext-holder">In This Section</h2>
+      </button>
+      <ul class="dg_tab-items">
+        <li class="active"><a href="/home">Newsroom</a></li>
+        <li class=""><a href="/home">Social Media Gallery</a></li>
```

Comment:
you shouldn't need these clases here, or maybe you want to do something like `dg_tab-items_item` so that you could style each li

Reply:
Removed the empty class and committed. I have a question regarding this item though. In my SubpageTabItem.js file, I have it setup as follows:
```
    <li className={TabItemClasses}>
      <a href={link}>{text}</a>
    </li>
```
Is there a way in the react file to setup if there is no class entered with TabItemClasses, it won't render an empty class="", or is there not really a way around that?

Reply:
@theilman7 looks like this is the way to do it, https://stackoverflow.com/a/48816255/1143670

But i wouldn't worry about it too much, I realized that i haven't really been doing my suggestion.